### ISSUE TITLE
Auto-laber For Project Ideas Github Issue

### ISSUE THREAD

**Issue by omenking** [100DaysOfCloud/100DaysOfCloudIdeas#4]
I think we'll need to label incoming project idea tickets by: Level, Category

Can someone look into this and implement it? How does this thing look?
https://github.com/apps/auto-labeler

@rishabkumar7 @johanrin

**Comment by omenking** [comment]
<img width="183" alt="Screen Shot 2020-07-07 at 10 08 36 AM" src="https://user-images.githubusercontent.com/7776/86793827-d9533800-c039-11ea-8d29-ffb85ee3db3f.png">

**Comment by antoniolofiego** [comment]
I can look into this.

**Comment by rishabkumar7** [comment]
@antoniolofiego you might wanna reach out to @johanrin since he said he was working on it.

**Comment by antoniolofiego** [comment]
Thanks for letting me know, @rishabkumar7. Will ask him!

**Comment by johanrin** [comment]
Someone to assign this issue to me?

**Comment by johanrin** [comment]
@omenking to give you an idea of what it looks like:

<img width="890" alt="Capture d’écran 2020-07-12 à 12 51 53" src="https://user-images.githubusercontent.com/13972080/87244594-9977c080-c43e-11ea-83eb-0d5bd768f3d8.png">

We can customize the color of the labels in the settings:

<img width="407" alt="Capture d’écran 2020-07-12 à 13 27 00" src="https://user-images.githubusercontent.com/13972080/87245149-64ba3800-c443-11ea-8198-93b9ba44b2f1.png">

To implement this, I used [issue-labeler Github Action](https://github.com/github/issue-labeler).
Basically, we define 2 YAML files and it will trigger a Github Action for each new issue.
1. `.github/labeler.yml` file with a list of labels and regex
2. `.github/workflows/labeler.yml` file to utilize the labeler action

Some questions:
- What is the actual template?
- Do I have a category (UNI, SEC, ...) in the body?

My Github Action is NOT able to detect the text in the title for now!
What it means, no category in the body = no category label

**Comment by omenking** [comment]
#### What is the actual template?
Template is here
https://github.com/100DaysOfCloud/100DaysOfCloudIdeas/blob/master/.github/ISSUE_TEMPLATE/cloud-project-idea.md

#### Do I have a category (UNI, SEC, ...) in the body?
They are not but we can add them no problem

**Comment by johanrin** [comment]
I edited the template and it worked for me.

Now, we have to define the colors because everything is gray... 😄

**Comment by omenking** [comment]
I see you added them good. I suppose I need to create the labels

**Comment by johanrin** [comment]
@omenking No need. The github action will create them automatically. We will adjust the color afterward.

**Comment by omenking** [comment]
I will close this ticket once I see a new project go through that has been tagged with the auto-label but I believe this ticket is done.

**Comment by what-name** [comment]
See #24 
Closed.
